### ISSUE TITLE
[BUG] Disk used amount does not update

### ISSUE THREAD

**Issue by wmww** [aristocratos/bashtop#39]
**Describe the bug**
When actual disk space is freed up, `Free` disk space is increased but `Used` space is not decreased (until bashtop is restarted).

**To Reproduce**
- Run bashtop
- Delete large file
- Observer that `Free` changed but `Used` did not, resulting in them not adding up to 100

**Expected behavior**
Both should be updated in real time

**Info (please complete the following information):**
 - Ubuntu 20.04
 - 5.0.16(1)-release

**Additional context**
$HOME/.config/bashtop/error.log:
```
New instance of bashtop version: 0.8.13 Pid: 1914836
 
New instance of bashtop version: 0.8.13 Pid: 1916971
 
New instance of bashtop version: 0.8.13 Pid: 1918707
```

**Comment by aristocratos** [comment]
Really thanks for finding this. The problem was the array for used disk space not being cleared, have been looking for what was eating up memory over time and this looks to be it!

**Comment by aristocratos** [comment]
Btw the disk and memory graphs are only updated every 10 updates, to keep cpu usage down. So it's not exactly realtime. Can lower it if it to slow?

**Comment by aristocratos** [comment]
Fixed in v0.8.14
Also increased memory and disk update frequency.
